expression/builtin_encryption.go

```diff
@@ -234,6 +234,10 @@ func (b *builtinPasswordSig) evalString(row types.Row) (d string, isNull bool, e
 		return "", false, nil
 	}
 
+	// Generate warning as the password function is deprecated as of MySQL 5.7.6
```

Comment:
maybe change the comment like this can be more readable:
```
// We should append a warning here because function "PASSWORD" is deprecated since MySQL 5.7.6.
```